src/components/ClaimableBalances.tsx

```diff
@@ -50,15 +71,17 @@ export const ClaimableBalances = () => {
   return (
     <div className="ClaimableBalances DataSection">
       <Layout.Inset>
-        <Heading2>Claimable Balances</Heading2>
+        <Heading2> Claimable Balance(s)</Heading2>
```

Comment:
It might be better to have the title dynamically show "Balance" vs "Balances", depending on what users have. What do you think?

Reply:
This is a simple yet very good idea. I have implemented the change as suggested, thank you